packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -389,18 +442,30 @@ export class AuroraPostgresEngineVersion {
    * @param auroraPostgresMajorVersion the major version of the engine,
    *   for example "9.6"
    */
-  public static of(auroraPostgresFullVersion: string, auroraPostgresMajorVersion: string): AuroraPostgresEngineVersion {
-    return new AuroraPostgresEngineVersion(auroraPostgresFullVersion, auroraPostgresMajorVersion);
+  public static of(auroraPostgresFullVersion: string, auroraPostgresMajorVersion: string,
+    auroraPostgresFeatures?: AuroraPostgresEngineFeatures): AuroraPostgresEngineVersion {
+
+    return new AuroraPostgresEngineVersion(auroraPostgresFullVersion, auroraPostgresMajorVersion, auroraPostgresFeatures);
   }
 
   /** The full version string, for example, "9.6.25.1". */
   public readonly auroraPostgresFullVersion: string;
   /** The major version of the engine, for example, "9.6". */
   public readonly auroraPostgresMajorVersion: string;
+  /**
+   * The supported features for the DB engine
+   *
+   * @internal
+   */
+  public readonly _features?: ClusterEngineFeatures;
```

Comment:
Actually, this is never `undefined`, right? So this should be `  public readonly _features: ClusterEngineFeatures;`, and that simplifies the code in `AuroraPostgresClusterEngine`.